src/main/java/it/reply/orchestrator/dto/request/DeploymentRequest.java

```diff
@@ -54,9 +54,14 @@
   private String callback;
 
   @Nullable
-  @Min(value = 1, message = "Timeout value, if provided, must be at least of 1 minute")
+  @Min(value = 1, message = "Overall timeout value, if provided, must be at least of 1 minute")
   private Integer timeoutMins;
 
+  @Nullable
+  @Min(value = 1, message = "Provider timeout value, if provided, must be at least of 1 minute and"
+      + " equal or less than timeoutMins")
+  private Integer providerTimeoutMins;
```

Comment:
Do you actually use this value?

Reply:
yeah!
    deploymentMessage.setProviderTimeoutInMins(request.getProviderTimeoutMins());

Reply:
And then where do you read it?

Reply:
you asked me to use the field in DeploymentRequest ... The one in DeploymentMessage is used by the workflow ... Didn't you see the workflow? <serviceTask id = "sid-D84A6907-D5D3-4908-8BC1-7C6A8A0B6870" name = "Handle Provider Timeout" flowable: async = "true" flowable: delegateExpression = "$ {providerTimeout}"> </serviceTask>

Reply:
> Didn't you see the workflow

can you point me at what line you use the value? because all I can see is that you're using a 3 minutes fixed value here https://github.com/indigo-dc/orchestrator/pull/372/files#diff-6a1bc734a9eec4532337628e6627ae29R66

Reply:
right, got lost in the various file update problems, fix it right away